index.html

```diff
@@ -1,13 +1,16 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="./src/assets/icons/mistborn.png" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Mistborn TTRPG Assistant</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/main.jsx"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="./src/assets/icons/mistborn.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Mistborn TTRPG Assistant</title>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="./src/main.jsx"></script>
```

Comment:
This diff is misleading because my linter indented everything. The only actual change that I make was on this line I added the dot in front of `./src/main.jsx`